docs/source/specs/openapi.json

```diff
@@ -426,6 +426,55 @@
                 }
             }
         },
+        "/organizations/aws/": {
+            "get": {
+                "tags": [
+                    "Organizations"
+                ],
+                "summary": "View AWS organizations",
+                "security": [{
+                    "basic_auth": []
+                }],
+                "parameters": [{
+                        "$ref": "#/components/parameters/QueryDelta"
+                    },
+                    {
+                        "$ref": "#/components/parameters/QueryFilter"
+                    },
+                    {
+                        "$ref": "#/components/parameters/QueryOffset"
+                    },
+                    {
+                        "$ref": "#/components/parameters/ReportQueryLimit"
+                    },
+                    {
+                        "$ref": "#/components/schemas/OrganizationFilter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "A paginated list of Organization objects",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/OrganizationPagination"
+                                }
+                            }
+                        }
+                    },
+                    "500": {
```

Comment:
I think you need to include the 403 response too

Reply:
I can add a 403 response. I didn't because I don't think that this endpoint returns a 403 unless you are actually adding a filter for an organization that you do not have access to (but it will never be returned without the filter). 

Also - we should probably go back and update the other endpoints because I don't see 403's anywhere in our openapispec and I suspect that other endpoints return them as well.

Reply:
@dccurtis added 😁